crates/bitwarden-api-api/src/models/cipher_details_response_model.rs

```diff
@@ -69,6 +69,8 @@ pub struct CipherDetailsResponseModel {
     pub view_password: Option<bool>,
     #[serde(rename = "collectionIds", skip_serializing_if = "Option::is_none")]
     pub collection_ids: Option<Vec<uuid::Uuid>>,
+    #[serde(rename = "archivedDate", skip_serializing_if = "Option::is_none")]
+    pub archived_date: Option<String>,
```

Comment:
This file is automatically generated from the server bindings and shouldn't be edited by hand: https://github.com/bitwarden/sdk-internal#api-bindings

I don't think you need to update this file at the moment, as we're not using the generated code anywhere yet. If you still want to update it, I recommend generating the bindings from server on a separate PR first.

Reply:
Thanks